spec/system/volunteer_adds_a_case_contact_spec.rb

```diff
@@ -125,7 +125,7 @@
   end
 
   context "with invalid inputs" do
-    it "re-renders the form with errors, but preserving all previously entered selections" do
+    xit "re-renders the form with errors, but preserving all previously entered selections" do
```

Comment:
is this a flaky test? :/ 
If so, please create a github issue "fix and reenable flaky test <testname>"